pkg/store/zookeeper.go

```diff
@@ -137,17 +137,17 @@ func (s *Zookeeper) CancelWatch(key string) error {
 }
 
 // GetRange gets a range of values at "directory"
-func (s *Zookeeper) GetRange(prefix string) (values [][]byte, err error) {
+func (s *Zookeeper) GetRange(prefix string) (kvi []KVEntry, err error) {
 	prefix = format(prefix)
-	entries, _, err := s.client.Children(prefix)
+	entries, stat, err := s.client.Children(prefix)
```

Comment:
Here we can probably rename `entries` to something else like `results` and change the return array with:
`GetRange(prefix string) (entries []KVEntry, err error)`

It'll be more meaningful with the now renamed struct.